memgpt/llm_api_tools.py

```diff
@@ -29,7 +29,8 @@ def is_context_overflow_error(exception):
 
     match_string = "maximum context length"
 
-    # Backwards compatability with openai python package/client v0.28 (pre-v1 client migration)
+    # Backwards compatibility with openai python package/client v0.28 (pre-v1 client migration)
+    # Backwards compatibility with openai python package/client v0.28 (pre-v1 client migration)

```

Comment:
Duplicate line

Reply:
Thanks a lot. Not even sure how that came in. Fixed now.